fastlane/Fastfile

```diff
@@ -549,17 +548,50 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each do |source, destination|
+      source_dir = File.join(Dir.pwd, "metadata", source)
+      Dir.children(source_dir).each { |file| 
+        source_file = File.join(source_dir, file)
+        destination_file = File.join(Dir.pwd, "metadata", destination, file)
+        FileUtils.mv(source_file, destination_file)
+      }
```

Comment:
Still worth using the `glob` suggestion that I mentioned in preview review, to avoid looping (esp given I think `children` includes `.` and `..` iirc?), and using `do…end` instead of `{…}`etc

Reply:
I actually ran into this issue which was why I used `Dir.children` 😅 

From https://ruby-doc.org/core-2.5.0/Dir.html#method-c-children:
> Returns an array containing all of the filenames except for “.” and “..” in the given directory. [...]

Also, it might just be that I'm very tired, but I'm not 100% sure what the `glob` suggestion was 😅 

One tricky thing – the `fastlane deliver download_metadata` action will put everything in the correct directory, but the `download_app_store_translations` lane still doesn't have everything that App Store Connect does. For that reason, I want to merge the directory contents instead of overwriting the directory. Not sure if that helps at all?

Sorry for the headache!!

Reply:
The glob suggestion is here: https://github.com/woocommerce/woocommerce-ios/pull/3352#discussion_r544471384

It still copy the dir content not the dir itself (thanks to the `*`), that's indeed a thing I kept in mind when I tried that code in `pry` to validate it before submitting the suggestion